tests/PHPUnit/System/expected/test_ImportLogs__PagePerformance.get_month.xml

```diff
@@ -6,5 +6,5 @@
 	<avg_time_dom_processing>0</avg_time_dom_processing>
 	<avg_time_dom_completion>0</avg_time_dom_completion>
 	<avg_time_on_load>0</avg_time_on_load>
-	<avg_page_load_time>0</avg_page_load_time>
+	<avg_page_load_time>0.13</avg_page_load_time>
```

Comment:
Shouldn't that be the same as `avg_time_server` as all other values are `0`? Maybe the `HAVING` doesn't work as expected though?